config/routes.rb

```diff
@@ -3,4 +3,12 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/customers/:id", to: "customers#show"
+
+  post "/customer_items", to: "customer_items#create"
```

Comment:
This looks great. See my comment on the url in your form for another way that you could write it.